lib/web_ui/lib/src/engine/compositor/canvaskit_api.dart

```diff
@@ -1537,3 +1611,41 @@ class SkFontMgrNamespace {
   // TODO(yjbanov): can this be made non-null? It returns null in our unit-tests right now.
   external SkFontMgr? FromData(List<Uint8List> fonts);
 }
+
+@JS()
+class SkData {
+  external int size();
+  external bool isEmpty();
+  external Uint8List bytes();
+}
+
+@JS()
+@anonymous
+class SkImageInfo {
+  external SkAlphaType get AlphaType;
+  external int get BitsPerPixel;
+  external int get BytesPerPixel;
+  external int get BytesSize;
+  external int get BytesSize64;
+  external SkColorSpace get ColorSpace;
+  external SkColorType get ColorType;
+  external int get Height;
+  external bool get IsEmpty;
+  external bool get IsOpaque;
+  external SkRect get Rect;
+  external int get RowBytes;
+  external int get RowBytes64;
+  // TODO external SkSize get Size;
```

Comment:
Avoid leaving commented out code. We can always add it when we need it. Otherwise there's risk that someone will add it below but also leave the TODO and it will linger here for a long time.